rule_monitor/Rule[] rules
int8 executing_rule_index